src/App.js

```diff
@@ -0,0 +1,144 @@
+import "./styles/main.scss";
+import Header from "./components/Header";
+import AddNewInput from "./components/AddNewInput";
+import Filter from "./components/FilterSortItems";
+import CheckListItem from "./components/CheckListItem";
+import React, { Component } from "react";
+
+const FILTER_MAP = {
+  All: () => true,
+  Active: (task) => !task.completed,
+  Completed: (task) => task.completed,
+};
+
+const FILTER_NAMES = Object.keys(FILTER_MAP);
+
+export default class App extends Component {
+  state = {
+    newTodo: "",
+    createdDate: "",
+    completed: false,
+    filter: "All",
+    isOldestFirst: true,
+    sortBy:"addedDate",
+    items: [
+      {
+        task: "go to shopping",
+        createdDate: "25 May 2021",
+        completed: true,
+      },
+    ],
+  };
+
+  handleOnChange = (event) => {
+    this.setState({ newTodo: event.target.value });
+  };
+  handleDate = (date) => {
+    this.setState({ createdDate: date.toDateString() });
+  };
+
+  handleOnSubmit = (event) => {
+    event.preventDefault();
+    this.setState({
+      items: [
+        ...this.state.items,
+        {
+          task: this.state.newTodo,
+          createdDate: this.state.createdDate,
+          completed: this.state.completed,
+        },
+      ],
+    });
+    this.setState({
+      newTodo: "",
+      createdDate: new Date(),
+    });
+  };
+
+  handleOnDelete = (index) => {
+    const items = this.state.items.filter((m, ind) => ind !== index);
```

Comment:
we should keep understandable variable in place of "m" and "ind"

please change to (item, index)